packages/office-ui-fabric-react/src/components/Pivot/Pivot.scss

```diff
@@ -7,233 +7,231 @@
-    }
+  &.isSelected {
+    color: $ms-color-themePrimary;
   }
 
-  // Ellipsis icon
-  .ms-Pivot-ellipsis {
-    font-size: $ms-font-size-m-plus;
-    position: relative;
-    top: 0;
+  &:hover:not(.isSelected),
+  &:focus:not(.isSelected) {
+    color: $ms-color-neutralDark;
   }
 
-  // Pivot content, hidden by default
-  .ms-Pivot-content {
-    display: none;
-    margin-top: 20px;
+  &:active {
+    color: $ms-color-themePrimary;
   }
+}
 
+// Ellipsis icon
+.ellipsis {
+  font-size: $ms-font-size-m-plus;
+  position: relative;
+  top: 0;
+}
 
-  //== Modifier: Large Pivots
-  //
-  .ms-Pivot.ms-Pivot--large {
-    .ms-Pivot-link {
-      font-size: $ms-font-size-l;
-    }
+// Pivot content, hidden by default
+.content {
+  display: none;
+  margin-top: 20px;
+}
 
-    .ms-Pivot-link.ms-Pivot-link--overflow {
-      &::after {
-        font-size: $ms-font-size-l;
-      }
+
+//== Modifier: Large Pivots
+//
+.root.isRootLarge {
+  .link {
+    font-size: $ms-font-size-l;
+  }
+
+  .link.isLinkOverflow {
+    &::after {
+      font-size: $ms-font-size-l;
     }
   }
+}
 
-  //== Modifier: Tabs
-  //
-  .ms-Pivot.ms-Pivot--tabs {
+//== Modifier: Tabs
+//
+.root.isRootTabs {
 
-    .ms-Pivot-link {
-      @include focus-border();
-      @include margin-right(0px);
-      height: 40px;
-      line-height: 40px;
-      background-color: $ms-color-neutralLighter;
-      padding: 0 10px;
-      vertical-align: top;
+  .link {
+    @include focus-border();
+    @include margin-right(0px);
+    height: 40px;
+    line-height: 40px;
+    background-color: $ms-color-neutralLighter;
+    padding: 0 10px;
+    vertical-align: top;
 
-      &:hover:not(.is-selected):not(.ms-Pivot-link--overflow),
-      &:focus:not(.is-selected):not(.ms-Pivot-link--overflow) {
-        color: $ms-color-black;
-      }
+    &:hover:not(.isSelected):not(.isLinkOverflow),
+    &:focus:not(.isSelected):not(.isLinkOverflow) {
+      color: $ms-color-black;
+    }
 
-      &:active {
-        color: $ms-color-white;
-        background-color: $ms-color-themePrimary;
-      }
+    &:active {
+      color: $ms-color-white;
+      background-color: $ms-color-themePrimary;
+    }
 
-      //== State: Selected
-      &.is-selected {
-        background-color: $ms-color-themePrimary;
-        color: $ms-color-white;
-        font-weight: $ms-font-weight-semilight;
+    //== State: Selected
+    &.isSelected {
+      background-color: $ms-color-themePrimary;
+      color: $ms-color-white;
+      font-weight: $ms-font-weight-semilight;
 
-        &::before {
-          background-color: transparent;
-          transition: none;
-        }
+      &::before {
+        background-color: transparent;
+        transition: none;
       }
     }
+  }
 
-    .ms-Pivot-link.ms-Pivot-link--overflow {
-      &:hover:not(.is-selected),
-      &:focus:not(.is-selected) {
-        background-color: $ms-color-white;
-      }
+  .link.isLinkOverflow {
+    &:hover:not(.isSelected),
```

Comment:
Which root class does the `isSelected` modifier apply to? Should be something like `linkIsSelected`